### ISSUE TITLE
[1.0.7] Unable to open forms in cp when reCAPTCHA is disabled

### ISSUE THREAD

**Issue by martinleveille** [solspace/craft-express-forms#46]
### Description

We are unable to open the /express-forms/forms in the cp when reCAPTCHA is disabled. Doing so will throw this exception: 

```
Argument 2 passed to Solspace\ExpressForms\objects\Form\Recaptcha::__construct() must be of the type string, null given, called in /Volumes/Data/Sites/test/vendor/solspace/craft-express-forms/src/decorators/Forms/Extras/RecaptchaDecorator.php on line 113
```

### Additional info

- Express Forms edition & version: 1.0.7 Lite
- Craft version: 3.5.12.1 Pro
- PHP version: 7.4.11
- Database driver & version: MySQL 5.7.31

**Comment by nfq** [comment]
I'm experiencing the same issue, latest version.

**Comment by nfq** [comment]
@kjmartens this is a major issue. I can't even access Express Forms in Craft to turn on reCAPTCHA. A couple clients have forms on their sites which are now inactive.

**Comment by kjmartens** [comment]
@martinleveille @nfq Sorry for the trouble. We've fixed this now in 1.0.8. 🙂

**Comment by nfq** [comment]
Thanks @kjmartens for the quick turnaround, really grateful.
